QMigrate/lut/create_lut.py

```diff
@@ -308,72 +343,66 @@ def _compute_1d_sweep(lut, vmod, kwargs):
 
     # Check nlloc_path is valid and contains Vel2Grid and Grid2Time
     if kwargs.get("nlloc_path", "") != "":
-        if not (nlloc_path / "Vel2Grid").exists() or not \
-            (nlloc_path / "Grid2Time").exists():
+        if not (nlloc_path / "Vel2Grid").exists() \
+           or not (nlloc_path / "Grid2Time").exists():
             raise FileNotFoundError(f"Incorrect nlloc_path? - Grid2Time and "
                                     f"Vel2Grid not found in {nlloc_path}")
 
-    lut.velocity_model = vmod
-    cc = lut.cell_count
-
-    # For NonLinLoc, distances must be in km
-    grid_xyz = [g / 1000 for g in lut.grid_xyz]
-    stations_xyz = lut.stations_xyz / 1000
-    ll, *_, ur = lut.grid_corners / 1000
+    # For NonLinLoc, distances/velocities must be in km - use conversion factor
+    km_cf = 1000 / lut.unit_conversion_factor
+    grid_xyz = [g / km_cf for g in lut.grid_xyz]
+    stations_xyz = lut.stations_xyz / km_cf
+    ll, *_, ur = lut.grid_corners / km_cf
+    nlloc_dx /= km_cf
+    vmodel = vmodel / km_cf
 
     # Make folders in which to run NonLinLoc
     cwd = pathlib.Path.cwd()
     (cwd / "time").mkdir(exist_ok=True)
     (cwd / "model").mkdir(exist_ok=True)
 
-    for i, station in lut.station_data.iterrows():
-        print(f"Computing 1-D sweep traveltime lookup table for station "
-              f"{station['Name']} - {i+1} of {stations_xyz.shape[0]}")
-
-        name = station["Name"]
-        lut.maps[name] = {}
+    for i, station in enumerate(lut.station_data["Name"].values):
+        logging.info(f"\t\t...running NonLinLoc - station: {station} - {i+1} "
```

Comment:
```suggestion
        logging.info(f"\t\t...running NonLinLoc - station: {station:5s} - {i+1} "
```

Minor formatting improvement.